Makefile

```diff
@@ -107,87 +114,121 @@ test:
 # Run generic integration tests
 .PHONY: test-generic
 test-generic:
+	odo login -u $(CLUSTER_USER_NAME) -p $(CLUSTER_PASSWORD) $(CLUSTER_LOGIN_URL) --insecure-skip-tls-verify
```

Comment:
FYI...
Due to open issue https://github.com/onsi/ginkgo/issues/440, i cannot make the ```odo login ...``` in suite level where login executed once globally before the spec.

Reply:
I think that makefile wouldn't ever contain `odo login`, or matter of fact, any other `odo` commands. 

Makefile should be environment agnostic.